src/doc/book/getting-started.md

```diff
@@ -24,41 +24,38 @@ see the website][platform-support].
 
 [platform-support]: https://forge.rust-lang.org/platform-support.html
 
-## Installing on Linux or Mac
+## Installing Rust
 
-If we're on Linux or a Mac, all we need to do is open a terminal and type this:
+All we need to do on On Unix systems like Linux and macOS is open a
+terminal and type this:
 
 ```bash
-$ curl -sSf https://static.rust-lang.org/rustup.sh | sh
+$ curl https://sh.rustup.rs -sSf | sh
 ```
 
-This will download a script, and start the installation. If it all goes well,
-you’ll see this appear:
+That will download a script, and start the installation. If it all
+goes well, you’ll see this appear:
 
 ```text
-Rust is ready to roll.
+Rust is installed now. Great! 
 ```
 
-From here, press `y` for ‘yes’, and then follow the rest of the prompts.
+Installing on Windows is nearly as easy: download and run
+[rustup-init.exe]. It will run the installation in a console and
+present the above message on success. For other installation options
+and information, visit the [install] page of the Rust website.
```

Comment:
> For other installation options and information, visit the [install] page of the Rust website.

Nit: It might make sense to put this on a different line than the Windows instructions.